README.md

```diff
@@ -49,6 +49,15 @@ test files can be executed via:
     ruby -Ilib:test test/FILENAME.rb
 
 
+### Usage
+
+These are methods
+
+    client.post(property)
```

Comment:
Was there a decision to not perform tasks on objects? My impression is the interface would look something like:

    Rentlinx.client.property(:property_id)
    # and/or
    Rentlinx::Property.from_id(:property_id)

Where both of those would return a lazy-loaded Rentlinx::Property class that contain `save`, and `delete` methods.

These are the similar bits of code in octokit corresponding to the "repository" type:
http://www.rubydoc.info/github/pengwynn/octokit/Octokit/Client/Repositories#repository-instance_method
http://www.rubydoc.info/github/pengwynn/octokit/Octokit/Repository#from_url-class_method